src/converter.js

```diff
@@ -157,6 +174,10 @@ function genValuePartial_toObject(gen, field, fieldIndex, prop) {
     if (field.resolvedType) {
         if (field.resolvedType instanceof Enum) gen
             ("d%s=o.enums===String?types[%i].values[m%s]:m%s", prop, fieldIndex, prop, prop);
+        else if (WRAPPER_TYPES.indexOf(field.type) !== -1) {
+          gen
+            ("d%s=types[%i].toObject(m%s,o).value", prop, fieldIndex, prop);
+        }
```

Comment:
Example difference in our output:

```
-                        d.creationTimestamp = $root.google.protobuf.UInt64Value.toObject(m.creationTimestamp, o);
+                        d.creationTimestamp = $root.google.protobuf.UInt64Value.toObject(m.creationTimestamp, o).value;
```

(See line 182 below to see what generated the previous version)